libraries/AP_Camera/AP_Camera_config.h

```diff
@@ -58,3 +58,7 @@
 #ifndef HAL_RUNCAM_ENABLED
 #define HAL_RUNCAM_ENABLED 1
 #endif
+
+#ifndef AP_CAMERA_RUNCAM_ENABLED
+#define AP_CAMERA_RUNCAM_ENABLED AP_CAMERA_BACKEND_DEFAULT_ENABLED && HAL_RUNCAM_ENABLED
```

Comment:
Any reason to keep `HAL_RUNCAM_ENABLED` around?

Reply:
makes it a bigger change to remove it. Would prefer to do a separate PR